### PR TITLE
jrsonnet: init at 0.3.8

### PR DESCRIPTION
**Pull request by CertainLach**
Rust implementation of jsonnet language

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -5653,6 +5653,16 @@
...
+  lach = {
+    email = "iam@lach.pw";
+    github = "CertainLach";
+    githubId = 6235312;
+    keys = [{
+      longkeyid = "rsa3072/40B5D6948143175F";
+      fingerprint = "323C 95B5 DBF7 2D74 8570  C0B7 40B5 D694 8143 175F";
+    }];
+    name = "Yaroslav Bolyukin";
+  };
```
Comment (legendofmiracles) [inline maintainers/maintainer-list.nix:5665]: separate commit: `maintainers: add lach`

File: pkgs/development/compilers/jrsonnet/default.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  src = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "CertainLach";
+    repo = "jrsonnet";
+    sha256 = "sha256-u6P/j7j6S7iPQQh00YFtp2G9Kt4xdWJGsxbuBjvHHZ4=";
+  };
+
+  postInstall = ''
+    ls target
```
Comment (legendofmiracles) [inline pkgs/development/compilers/jrsonnet/default.nix:15]:
```suggestion
```
this serves no purpose

File: pkgs/development/compilers/jrsonnet/default.nix [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  '';
+
+  cargoSha256 = "sha256-KGQ3n3BBgLCT3ITIM8p9AxNa62ek4GHymqoD0eQSVKQ=";
+
+  meta = {
+    description = "Purely-functional configuration language that helps you define JSON data";
+    maintainers = with lib.maintainers; [ lach ];
+    license = lib.licenses.mit;
+    homepage = "https://github.com/CertainLach/jrsonnet";
+    platforms = lib.platforms.all;
```
Comment (legendofmiracles) [inline pkgs/development/compilers/jrsonnet/default.nix:26]:
```suggestion
```
set by rust

**Comment by ethancedwards8** [comment]
Result of `nixpkgs-review pr 127401` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>jrsonnet</li>
  </ul>
</details>

**Pull request merged**